app/src/main/java/cm/aptoide/pt/UpdateLaunchManager.kt

```diff
@@ -0,0 +1,17 @@
+package cm.aptoide.pt
+
+import rx.Completable
+
+class UpdateLaunchManager(val followedStoresManager: FollowedStoresManager) {
+
```

Comment:
Current version code not being used

Reply:
I can remove but I did it on purpose because there might be specific scenarios where you only want to do a change coming from a specific version to another specific version. I think it's generally standard to pass the previous and the current version in these type of things.